packages/hash/api/src/graphql/typeDefs/knowledge/page.typedef.ts

```diff
@@ -1,7 +1,7 @@
 import { gql } from "apollo-server-express";
 
 export const persistedPageTypedef = gql`
-  type PersistedPage implements PersistedEntity {
+  type PersistedPage implements EntityWithMetadata {
```

Comment:
Do we still need to implement?

Reply:
For now, yes. But hopefully we can get rid of this when frontend relies on subgraphs. Only reason why this is kept is so that pages and comments would not need a full-on frontend rewrite yet.

Reply:
I more meant that can we still keep their structure (with the interface fields 🙃) but not actually `implements`. If we're using the interface fair enough.

Reply:
I'll give it another look and check these relationships. Maybe we can get rid of `UnknownEntity`

Reply:
Would be fantastic

Reply:
I got rid of unknown entirely in 61cd88133749f9c94709511cc2460d2925a27aca
This only works because we don't rely on the polymorphism of types anymore since we introduced Subgraph for all the parts of the queries that would require being able to tell what type we're dealing with